src/config.py

```diff
@@ -0,0 +1,16 @@
+from os import getenv
+
+debug = True
+
+if debug:
+    from dotenv import load_dotenv
+
+    load_dotenv(override=True)
+
+
+token = getenv("TOKEN")
+suggestion_channel = int(getenv("SUGGESTION_CHANNEL"))
+role1 = int(getenv("ROLE1"))
+role2 = int(getenv("ROLE2"))
```

Comment:
also this?

Reply:
because it was using roles: https://github.com/MaskDuck/impostor-helper-bot/pull/3/files#diff-12180a970eb086eaebbfdc0dfa5a97eb4cf99b1db0c92ede6a843ccb4ff3073eL11

Reply:
it would make more sense to use the name of roles (example `maintainer_role`) instead of `role` and `role2`.

Reply:
Changed